aks/nginx-ingress.tf

```diff
@@ -26,5 +26,5 @@ resource "helm_release" "nginx_ingress" {
       value = "true"
   }
 
-  depends_on = ["helm_release.external-dns"]
+  depends_on = [helm_release.external-dns]
```

Comment:
can we add a newline at the end of this file.

Reply:
Done.